src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -23,11 +23,12 @@
     [angularticsLabel]="item.label"
     *ngFor="let item of menuItems"
     [routerLink]="[item.link]"
+    [class.primary-background]="item.link === activeLink"
```

Comment:
When visiting the `Profile`, the Dashboard is marked as the active menu item.
![image](https://user-images.githubusercontent.com/33730997/150767035-5f665610-3cb0-481b-bd9e-dcf11a1026a7.png)

Reply:
Suggesting to add a fallback in the menu highlighting: for root path (`/`) do only highlight for exact match.